src/components/NavBar/index.js

```diff
@@ -48,9 +47,9 @@ class NavBar extends React.Component {
   }
 
   componentDidMount() {
-    const userClaims = this.context;
+    const { userRoles } = this.context;
     this.setState({
-      isOfficer: checkIsOfficer(userClaims),
+      isOfficer: userRoles.includes('officer'),
```

Comment:
i'd prefer to see a function that takes in context and tells us the role - that way the components aren't as tightly coupled with how userRoles are stored within context.

Reply:
or rather tells us whether or not a "context" has a role e.g. `isOfficer(userContext)`

Reply:
I can put functions like isOfficer(userContext), isMember(userContext) and isInductee(userContext) in the new services/claims.js. I'm guessing userContext is just this.context (like in NavBar/index.js) right?